index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lawffice</title>
+</head>
+<body>
+    <header><h1>Law Office of Maisie A. Barringer</h1></header>
+    <nav>
+        <ul>
```

Comment:
this list needs to be in the header section if we're putting it all on the same single line bar at the top.